wallet/wallet.go

```diff
@@ -3875,7 +3878,7 @@ func (w *Wallet) TotalReceivedForAddr(addr dcrutil.Address, minConf int32) (dcru
 
 // SendOutputs creates and sends payment transactions. It returns the
 // transaction hash upon success
-func (w *Wallet) SendOutputs(outputs []*wire.TxOut, account uint32, minconf int32) (*chainhash.Hash, error) {
+func (w *Wallet) SendOutputs(outputs []*wire.TxOut, account uint32, minconf int32, recipientPaysFee bool) (*chainhash.Hash, error) {
```

Comment:
This is a breaking API change. You will either need to bump the wallet's major module version (a lot of work) or find a backwards-compatible way to introduce this feature into the public API.

Reply:
fixed, I think?